### ISSUE TITLE
rgba() support for cleanupListOfValues 

### ISSUE THREAD

**Issue by peternowell** [svg/svgo#628]
It would be nice if _cleanupListOfValues_ could work on `rgba()` values... removing unnecessary spaces and leading zeros. Not sure if rounding to the specified precision would be necessary, but it would be nice if `rgba(10, 20, 30, 0.4)` could turn into `rgba(10,20,30,.4)` .

**Comment by strarsis** [comment]
Isn't this CSS? The [minifyStyles plugin](https://github.com/svg/svgo/blob/master/plugins/minifyStyles.js) uses csso which also minifies color values like `rgba(...)`.

**Comment by GreLI** [comment]
Yep, it's defined in CSS Colors Module level 3 and many browser supports it, but it wasn't defined in SVG 1 and thus not supported in SVGO.

**Comment by peternowell** [comment]
So does that mean that the SVGO's CSS colors minifier couldn't be augmented with this feature? Given that there are a very limited number of color formats in this scenario, couldn't this just be written from scratch pretty quickly? Just remove spaces, round numbers to specified level of precision, and remove any zeros preceeding decimals?

**Comment by strarsis** [comment]
@GreLI: So plain CSS is optimized by CSSO in minifyStyles plugin. - 
But rgba values in element attributes are not supported currently?

**Comment by GreLI** [comment]
@peternowell, it totally, I just described reason why it doesn't now. 

@strarsis, it doesn't supported for now in SVGO. What processed through CSSO depends on CSSO support (I guess it does.)

**Comment by strarsis** [comment]
@peternowell : But because CSSO handles the CSS minification, it already minifies rgba values. 
This issue can be closed then?

**Comment by GreLI** [comment]
I believe the issue is about rgba values outside of CSS or `style` attribute.

**Comment by strarsis** [comment]
It is possible reusing the CSSO routines, also for other attribute values than just for color (https://github.com/css/csso/tree/f381f09436d66572df0ccbca45c39aa84709269f/lib/replace).

For color values, CSSO also replaces generic colors with their keywords (which is usually also more efficient), see https://github.com/css/csso/blob/f381f09436d66572df0ccbca45c39aa84709269f/lib/replace/color.js .
